servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/updatePayment.js

```diff
@@ -80,7 +80,13 @@ module.exports = async (_, args, {pgdb, req, t}) => {
         })
       }
 
-      if (pledge.total > 100000) {
+      const hasPledgeMemberships = await pgdb.public.memberships.count({
+        pledgeId: pledge.id
+      })
+
+      // Only generate memberships (or periods) of pledge has not generated
+      // memberships already.
+      if (hasPledgeMemberships < 1) {
```

Comment:
This should "fix" `updatePayment` mutation:
- Generate memberships if pledge did not cause any memberships this far e.g. `ABO`
- Generate periods (via `generateMemberships`) if they were not created yet e.g. `PROLONG`